docs/src/introduction.md

```diff
@@ -58,6 +59,32 @@ p4est_connectivity_destroy(connectivity)
 ```
 
 
+## [`PointerWrapper`s](@id pointer_wrappers)
+
+As we have seen in the [previous section](@ref translation_guidelines) many functions
+provided by [`p4est`](https://github.com/cburstedde/p4est) return pointers to `struct`s,
+which requires to [`unsafe_load`](https://docs.julialang.org/en/v1/base/c/#Base.unsafe_load)
+the pointer in order to access the underlying data.
```

Comment:
```suggestion
which requires one to [`unsafe_load`](https://docs.julialang.org/en/v1/base/c/#Base.unsafe_load)
the pointer in order to access the underlying data.
```
"require" is a transitive verb and thus requires a direct object (no pun intended)